### ISSUE TITLE
Add "commit --fixup" option into the Repositories view

### ISSUE THREAD

**Issue by lmeneguzzi** [gitkraken/vscode-gitlens#1031]
Hi, I want to suggest a simple feature that will be very useful to me and maybe can be useful to other users too.

My suggestion is to have an option on a right-click in a commit to execute the command `git commit --fixup=<commit_id>` using the commit id that you click and committing the changes that are staged, this option will be in the `Repository view`.
![image](https://user-images.githubusercontent.com/58669455/82014276-a781ae80-9652-11ea-8032-41890ba1506d.png)

Basically the option will be together with the other options like `Revert Commit...`, etc.
Or can be done a shortcut and stay together with the `Copy Commit ID` too:
![image](https://user-images.githubusercontent.com/58669455/82014732-96856d00-9653-11ea-9d8b-b7a8d1dc1ac2.png)

**Comment by eamodio** [comment]
Sounds like a good idea. Would you mind opening a PR to add it or get it started? I'd be happy to help point in the right direction, etc.

**Comment by yuichkun** [comment]
@eamodio 
Hi, I'm interested in implementing this!

I started looking at the code, messed around, and patched things together in a **really dirty way** (a lot of copying and pasting without fully understanding), but it seems kind of working now on my local mac 🤔 

If I create a draft PR for this, would you maybe take a look at it and give me some hints??
I'd be more than happy to revise my code until it meets your criteria!

**Comment by elupus** [comment]
It would be cool to get support for instant fixup. That is git commit --fixup && git stash && git rebase --autosquash commit^ && git stash apply. With some error checking.

**Comment by bam80** [comment]
Progress?

**Comment by d13** [comment]
We've added this feature in the Commit Graph, see: https://help.gitkraken.com/gitlens/gitlens-release-notes-current/#fixup-commits-autosquash-and-push-tags

Resolved with 16d8901c6
